spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -660,6 +660,10 @@ definitions:
             type: double
             units: m
             desc: Height above WGS84 ellipsoid
+        - orthometric_height:
```

Comment:
Should we be deprecating the original definition and creating a new version of this message with the extra field?

Reply:
Answered on slack